src/ws/test-auth.c

```diff
@@ -1175,7 +1172,8 @@ main (int argc,
 {
   cockpit_ws_session_program = BUILDDIR "/mock-auth-command";
   cockpit_ws_service_idle = 1;
-  cockpit_ws_process_idle = 2;
+
+  g_assert (g_setenv ("COCKPIT_WS_PROCESS_IDLE", "2", TRUE));
```

Comment:
I know this is a test, but assert() on an expression with desired side effects has always bothered me.

Reply:
Fixed.